test/asynch_spi/asynch_spi.cpp

```diff
@@ -58,7 +58,12 @@ class SPITest {
         printf("Starting short transfer test\r\n");
         init_rx_buffer();
         cs = 0;
-        printf("Res is %d\r\n", spi.transfer(tx_buf, SHORT_XFR, rx_buf, SHORT_XFR, SPI::event_callback_t(this, &SPITest::short_transfer_complete_cb), SPI_EVENT_COMPLETE));
+        int rc = spi.transfer()
+            .tx(tx_buf, SHORT_XFR)
+            .rx(rx_buf, SHORT_XFR)
+            .callback(SPI::event_callback_t(this, &SPITest::short_transfer_complete_cb), SPI_EVENT_COMPLETE)
+            .apply();
```

Comment:
```apply()``` allows you to reap status output.  The other option is to provide a cast-to-int.